README.md

```diff
@@ -163,6 +163,19 @@ applications into the project.
 
 [buildpack]: https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app#adding-a-buildpack
 
+## Capistrano
+EmberCLI-Rails executes both `npm install` and `bower install` as part of the asset precompilation step of a Capistrano build. This requires that the correct version of these executables be found in the path of your deployment SSH session. Since Capistrano executes in a very basic shell, it is not sufficient to place the PATH in your `.bash_profile`. 
+
+Adding the bin directory of your node install to your capistrano deploy script will enable EmberCLI-Rails to find the correct bower and npm executables.
+
+```ruby
+#config/deploy/production.rb
+set :default_env, {
+  'PATH' => '/home/deploy/.nvm/versions/node/v4.2.1/bin:$PATH' #note: using NVM to set node version in this example. Replace with output of 'which npm' in your server environment.
```

Comment:
It depends on your node configuration. If you are using the system node, ‘which npm’ returns /usr/bin/npm.  If you are using NVM node version manager, it will return the bin directory from the in-use version (as in my example). 

You need to remove /npm from the answer to ‘which npm’ to get the correct directory for the path.  And bower needs to be installed under that version of node as well. 

I agree this may not be the clearest explanation — perhaps saying: 

1) Make sure you have selected the production version of Node that your server uses (i.e. via nvm use xxx, or just installing it as system).
2) Execute npm install -g bower 
3) Take the path from these (i.e. ‘which bower’ or ‘which npm’) and make sure they are first in the PATH used by capistrano, below.


What do you think?

> On Dec 10, 2015, at 4:33 PM, Sean Doyle <notifications@github.com> wrote:
> 
> In README.md <https://github.com/thoughtbot/ember-cli-rails/pull/350#discussion_r47289207>:
> 
> > @@ -163,6 +163,19 @@ applications into the project.
> >  
> >  [buildpack]: https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app#adding-a-buildpack
> >  
> > +## Capistrano
> > +EmberCLI-Rails executes both `npm install` and `bower install` as part of the asset precompilation step of a Capistrano build. This requires that the correct version of these executables be found in the path of your deployment SSH session. Since Capistrano executes in a very basic shell, it is not sufficient to place the PATH in your `.bash_profile`. 
> > +
> > +Adding the bin directory of your node install to your capistrano deploy script will enable EmberCLI-Rails to find the correct bower and npm executables.
> > +
> > +```ruby
> > +#config/deploy/production.rb
> > +set :default_env, {
> > +  'PATH' => '/home/deploy/.nvm/versions/node/v4.2.1/bin:$PATH' #note: using NVM to set node version in this example. Replace with output of 'which npm' in your server environment.
> Replace with output of 'which npm' in your server environment.
> 
> will the output of which npm be swappable with /home/deploy/.nvm/versions/node/v4.2.1/?
> 
> Is there another command that will produce the proper path?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/ember-cli-rails/pull/350/files#r47289207>.
>